dotcom-rendering/src/components/StickyBottomBanner.importable.tsx

```diff
@@ -116,6 +118,7 @@ const buildRRBannerConfigWith = ({
 		tags: TagType[];
 		contributionsServiceUrl: string;
 		idApiUrl: string;
+		isAndroidWebview: boolean | undefined;
```

Comment:
just a small note to say that it may be confusing to have to think about the union type `true | false | undefined` and it might be better to coerce to a boolean upstream!

Reply:
👍